core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2196,6 +2220,15 @@ public String toString() {
         "alluxio.security.group.mapping.class";
     public static final String SECURITY_LOGIN_USERNAME = "alluxio.security.login.username";
 
+    //
+    // JVM related properties
+    //
+    public static final String JVM_MONITOR_WARN_THRESHOLD_MS = "jvm.monitor.warn.threshold.ms";
+    public static final String JVM_MONITOR_INFO_THRESHOLD_MS = "jvm.monitor.info.threshold.ms";
+    public static final String JVM_MONITOR_SLEEP_INTERVAL_MS = "jvm.monitor.sleep.interval.ms";
+    public static final String JVM_MONITOR_MASTER_START = "jvm.monitor.master.start";
```

Comment:
rename to `jvm.monitor.master.enable`

Reply:
Also please update the configurations referenced in this doc https://github.com/Alluxio/alluxio/blob/master/docs/en/Configuration-Properties.md